package.json

```diff
@@ -13,6 +13,7 @@
     "prepublish": "npm run build"
   },
   "dependencies": {
+    "babelify": "^7.2.0",
```

Comment:
For the build, it doesn't work without it. it replaces some babel lib that was deleted